projects/NeRF-Det/nerfdet/det3d_data_sample.py

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+from mmdet.structures import DetDataSample
+from mmengine.structures import InstanceData
+
+from mmdet3d.structures import PointData
+
+
+class Det3DDataSample(DetDataSample):
+    """A data structure interface of MMDetection3D. They are used as interfaces
```

Comment:
Rename it with `NerfDet3DDataSample` and inheirt with `Det3DDataSample`. Explain its difference with default DataSample